Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.h

```diff
@@ -61,31 +61,31 @@ typedef void(^VOKManagedObjectsReturnBlock)(VOKArrayOfManagedObjects *managedObj
  Create or update many NSManagedObjects, respecting the mapper's overwriteObjectsWithServerChanges and ignoreNullValueOverwrites properties.
  This should only be used to set all properties of an entity. 
  By default any mapped attributes not included in the input dictionaries will be set to nil.
- This will overwrite ALL of an NSManagedObject's properties unless ignoreNullValueOverwrites is YES;
+ This will overwrite ALL of an NSManagedObject's properties unless ignoreNullValueOverwrites is YES on the mapper for this class.
  @param inputArray      An array of dictionaries with foreign data to inport.
- @param contextOfNil    The managed object context to update and/or insert the objects. If nil, the main context will be used.
+ @param contextOfNil    The managed object context in which to update and/or insert the objects. If nil, the main context will be used.
  @return                An array of this subclass of NSManagedObject.
  **/
 + (VOKArrayOfManagedObjects *)vok_addWithArray:(VOKArrayOfObjectDictionaries *)inputArray
                        forManagedObjectContext:(nullable NSManagedObjectContext *)contextOrNil;
 
 /*
- Create or update a NSManagedObject, respecting the mapper's overwriteObjectsWithServerChanges and ignoreNullValueOverwrites properties.
+ Create or update a single NSManagedObject, respecting the mapper's overwriteObjectsWithServerChanges and ignoreNullValueOverwrites properties.
  This should only be used to set all properties of an entity.
  By default any mapped attributes not included in the input dictionaries will be set to nil.
- This will overwrite ALL of an NSManagedObject's properties unless ignoreNullValueOverwrites is YES;
+ This will overwrite ALL of an NSManagedObject's properties unless ignoreNullValueOverwrites is YES on the mapper for this class.
  @param inputDict       A dictionary with foreign data to inport.
- @param contextOfNil    The managed object context to update and/or insert the object. If nil, the main context will be used.
+ @param contextOfNil    The managed object context in which to update and/or insert the object. If nil, the main context will be used.
  @return                An instance of this subclass of NSManagedObject.
  **/
 + (nullable instancetype)vok_addWithDictionary:(VOKStringToObjectDictionary *)inputDict
                        forManagedObjectContext:(nullable NSManagedObjectContext *)contextOrNil;
 
 /*
- Create or update many NSManagedObjects, respecting the mapper's overwriteObjectsWithServerChanges and ignoreNullValueOverwrites properties.
+ Create or update many NSManagedObjects in a background managed object context, respecting the mapper's overwriteObjectsWithServerChanges and ignoreNullValueOverwrites properties.
```

Comment:
How about "on a background operation queue"?  I'm worried that "background managed object context" is unclear/inaccurate.